README.md

```diff
@@ -1,101 +1,98 @@
   <br>
   <br>
-  <a href="https://github.com/AxonIQ/code-samples">Code Samples Repo</a>
+  <a href="https://www.axoniq.io/products/axon-framework">Axon Framework</a>
   ·
-  <a href="https://developer.axoniq.io/axon-framework/overview">Technical Overview</a>
+  <a href="https://www.axoniq.io/products/axon-server">Axon Server</a>
   ·
-  <a href="https://github.com/AxonFramework/AxonFramework/issues">Feature / Bug Request</a>
+  <a href="https://www.axoniq.io/products/axoniq-console">AxonIQ Console</a>
 
 </p>
 
+#
 
+<p><br/></p>
+<img src="https://www.axoniq.io/hubfs/axon-framework-line-light.png" alt="Axon Framework logo" width="500">
 
+The Axon Framework is an open source framework that's 100% Java and enables developers to build scalable and maintainable applications using a message-driven approach. It simplifies the complexities of developing distributed systems by providing a structured way to handle commands, events, and queries within your application. [If you're not familiar with the Event Sourcing pattern](https://www.axoniq.io/concepts/cqrs-and-event-sourcing), you can learn more about it on our website.
 
-# Axon Framework
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.axonframework/axon/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.axonframework/axon)
-![Build Status](https://github.com/AxonFramework/AxonFramework/workflows/Axon%20Framework/badge.svg?branch=master)
-[![SonarCloud Status](https://sonarcloud.io/api/project_badges/measure?project=AxonFramework_AxonFramework&metric=alert_status)](https://sonarcloud.io/dashboard?id=AxonFramework_AxonFramework)
+At its core, the Axon Framework encourages an architecture where components communicate through messages, which promotes loose coupling and increases flexibility. This messaging system allows different parts of your application to interact without needing to know the internal workings of each other, making your system more modular and easier to manage. If you already use tools and technologies that enable your applications to utilize a pub/sub pattern, then you already understand the basics of message driven and event driven architectures.
 
-Axon Framework is a framework for building evolutionary, event-driven microservice systems based on the principles of Domain-Driven Design (DDD), Command-Query Responsibility Separation (CQRS), and Event Sourcing.
+The Axon Framework further enhances your applications and services by enabling you to support for Event Sourcing, a powerful architectural pattern where state changes are recorded as a sequence of events. Why is this important? Well, instead of just storing the current state in a traditional database, every change is logged, providing a complete history of how the data arrived at its current form. This can be incredibly useful for auditing, debugging, and even recreating past states of your application when necessary.
```

Comment:
The part "...services by enabling you to support for Event Sourcing..." feels weird as a sentence. Shouldn't `for` be removed from this sentence?